liferay-tango/liferay-portal
# LPS-151362-Variation Add segmentation disable warning for Variations

Motivation
=======
As a Marketer, I want to be warned in context when Segmentation is disabled, so that I am aware that my Segments, Experiences and Collections will not function properly without this configuration enabled.

[Link to story or ticket](https://issues.liferay.com/browse/LPS-151362)

Proposed Solution
========
The proposed solution consists in adding the keywords needed for being able to add a warning with the correct message in Variations(Collections) and add the necessary code to render it.

Steps to Verify:
----------------
1. Go to Menu -> Control Panel -> Instance Settings -> Segments
2. In Segments click on Segments Service -> Enable Segmentation (disable it if the box is checked)
3. Go to Site Builder -> Collections
4. Add a Manual Collection and open it
5. Check that an Alert Warning message is displayed correctly
6. Repeat step 4-5 with Dynamic Collection
---------------

Screenshots (optional):
-----------------------

<img width="1440" alt="Screenshot 2022-05-23 at 18 32 40" src="https://user-images.githubusercontent.com/91207948/169866459-76378c19-d8bd-4d40-94ac-f0dc27657436.png">
<img width="1440" alt="Screenshot 2022-05-23 at 18 32 23" src="https://user-images.githubusercontent.com/91207948/169866468-e901248d-95d9-4de6-9239-f28bdd6e7da8.png">
